lms/envs/production.py

```diff
@@ -157,6 +157,12 @@ def get_env_setting(setting):
 DCS_SESSION_COOKIE_SAMESITE = ENV_TOKENS.get('DCS_SESSION_COOKIE_SAMESITE', DCS_SESSION_COOKIE_SAMESITE)
 DCS_SESSION_COOKIE_SAMESITE_FORCE_ALL = ENV_TOKENS.get('DCS_SESSION_COOKIE_SAMESITE_FORCE_ALL', DCS_SESSION_COOKIE_SAMESITE_FORCE_ALL)  # lint-amnesty, pylint: disable=line-too-long
 
+# As django-cookies-samesite package is set to be removed from base requirements when we upgrade to Django 3.2,
+# we should follow the settings name provided by Django.
+# https://docs.djangoproject.com/en/3.2/ref/settings/#session-cookie-samesite
+if django.VERSION >= (3, 1):
+    SESSION_COOKIE_SAMESITE = DCS_SESSION_COOKIE_SAMESITE
```

Comment:
We found this issue when we were testing on the sandbox with Django 3.2. When we tried to log in with Apple ID, it broke, and on investigation found that `SESSION_COOKIE_SAMESITE` was set `'Lax'`. `'Lax'` is the default value for this setting, but we wanted the value to be `'None'` which was introduced in Django 3.1 (*).

So for now we have added a condition to set the value. After this upgrade, we will replace `DCS_SESSION_COOKIE_SAMESITE` completely with Django's provided settings name. 

(*) https://docs.djangoproject.com/en/3.2/ref/settings/#session-cookie-samesite